Makefile

```diff
@@ -323,8 +321,8 @@ SVG/offline/index.html: scripts/make_svg_index.pl scripts/svg.css $(SVGS)
 	$< $(filter %.svg,$^) > $@
 	ln -f scripts/svg.css $(@D)/
 
-MIDI/%.midi: LYOPTS += --define-default=include-settings=variants/MIDI-settings.ily
 MIDI/%.midi: LYOPTS += --define-default=no-print-pages
+MIDI/%.midi: LYOPTS += --define-default=eog-midi-permitted
```

Comment:
This line yields a warning since `eog-midi-permitted` is not a standard setting. Maybe we can find a better way to propagate this setting, if we still need it.

Reply:
This works well enough for now; I may revisit this in a future set of changes.